test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/reliefdetails/ReliefDetailsControllerSpec.scala

```diff
@@ -182,7 +182,13 @@ class ReliefDetailsControllerSpec
       }
 
     }
+    "handling check your answers" must {
```

Comment:
there should already be a section in the spec for the check your answers page. Within that section is already an area where it's testing when we should be redirected to the other reliefs page (see L1188) - these tests belong there. I'd take a look at those tests to see how it's being done. Also, you don't need to have an form data in the body for check your answers

Reply:
Yes. I think I was in a rush yesterday to get this done to move on with 1252, that is why I missed that.. Done.